src/typings.ts

```diff
@@ -81,7 +81,7 @@ export interface IMetadataHolder<VALUES extends AllowedMetadataValueTypes = stri
 
 export type AllowedMetadataValueTypes = string | boolean | number | undefined;
 
-export type MangaMetadataKeys = keyof IReaderSettings;
+export type MangaMetadataKeys = keyof (IReaderSettings & ISearchSettings);
```

Comment:
AppMetadataKeys should be a better place to add the type, since the search settings aren't really part of the mangas